protos/ensembl/production/metadata/grpc/ensembl_metadata.proto

```diff
@@ -46,6 +46,9 @@ service EnsemblMetadata {
   // Retrieve genomes by keyword search
   rpc GetGenomesBySpecificKeyword(GenomeBySpecificKeywordRequest) returns (stream Genome) {}
 
+  // Retrieve genomes by release version
+  rpc GetGenomesByReleaseVersion(GenomeByReleaseVersionRequest) returns (stream Genome) {}
```

Comment:
@darefalola , I also realised that we don't need all the detailed returned by `Genome`, `BriefGenomeDetails` should be more than enough
Can you you update the code please, this will speed up the response time
Thanks